docs/content/topics/project-developers/brig.md

```diff
@@ -1,6 +1,5 @@
 ---
-linkTitle: Using the brig CLI
-title: Brig
+title: The brig CLI
```

Comment:
Didn't think the title here provided enough context. Thought the link title was a bit long. Met in the middle. 😄